CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.10.0 (July 27, 2023)
+
+DOCUMENTATION:
+* Rebrand to Turbot Guardrails provider. ([#159](https://github.com/turbot/terraform-provider-turbot/issues/159))
```

Comment:
Suggest we highlight here that no change is required for user configs etc. (Don't scare them.)